docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,52 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.ff
+
+This guide provides the information of
+
+- To create a service account in vendasta and authorization token to access vendasta APIs
+- List of Vendasta SCIM APIs and the way to use it.
+
+
+## Step 1 : Pre-requisites for accessing the Vendasta's SCIM APIs
+
+### 1. Namespace
+
+This must be the id of the partner that wish to manage users in sync with Vendasta. It should be unique for a partner.
+
+The users access Partner Center is that when a new Channel Partner signs up a new Partner is created and this generates a PID. The PID is then set as the [namespace](https://vendasta.jira.com/wiki/spaces/RD/pages/1841005358/Business+App+Needs+a+Namespace) of all the SCIM Users. 
+
+
+![image.png](../../../assets/images/image-2.png)
+
+
+### 2. Authorization token
+- For a partner with Vendasta, in order to access APIs of Vendasta they must need a authorisation token which is generated against the namespace of the partner. This should be refreshed over a particular time.
+
+- The authorization token must be generated with required scopes in order to access a particular platform. Here "user.admin" is the required scope which is mandatory for each token. 
+
+
+> To create a service account and create a token , see [here](../../Authorization/2-legged-oauth/Overview.md).
+
+
+
+## Step 2 : Vendasta SCIM Endpoints to sync users
+
+A [schema](https://www.ibm.com/docs/en/odi?topic=apis-schema-api) is metadata that tells us how our data is structured. Most databases implement some form of schema which enables us to reason about our data in a more structured manner. 
+We support a [User schema](../../../openapi/scim/scim.yaml/paths/~1{namespace}~1Schemas) which is used in our SCIM APIs.
```

Comment:
- Include [List Schemas](https://developers.vendasta.com/platform/916d64f3cd829-list-schemas) which can be used to discover schema
- Include the [System Operation](https://developers.vendasta.com/platform/f45a171a7f52f-list-resource-types) section which will expose all of our supported configurations